pronotepy/dataClasses.py

```diff
@@ -318,9 +319,69 @@ def __init__(self, parsed_json: dict) -> None:
         self.name: str = self._resolver(str, "L")
         self.groups: bool = self._resolver(bool, "estServiceGroupe", default=False)
 
+
+class ReportSubject(Subject):
```

Comment:
This could probably be an inner class of `Report`, since it probably won't see any use outside this specific context.